lib/new_relic/agent/method_tracer_helpers.rb

```diff
@@ -6,6 +6,8 @@ module NewRelic
   module Agent
     module MethodTracerHelpers
       MAX_ALLOWED_METRIC_DURATION = 1_000_000_000 # roughly 31 years
+      SOURCE_CODE_INFORMATION_PARAMETERS = %i[filepath lineno function namespace]
```

Comment:
I wonder if we should add a comment about the use of the word "function"? I itch to change it to "method", but know it's there because of the specs.

Reply:
Alternately, it could be called method except for the final key? https://github.com/newrelic/newrelic-ruby-agent/pull/1069/files#diff-1bf0a4e80d440cbe0b83bfe60e1b97e75696a89369dffaacbc442cd16ff0a6a1R134

Reply:
I really wanted to retain the 4 variable names from the spec, but I liked the commenting idea. Added comments with 6ff053b.